sample/ios/www/jxcore/thali/mockmobile.js

```diff
@@ -0,0 +1,58 @@
+var keys = {};
+
+function Mobile(key) {
+  keys[key] || (keys[key] = new NativeCall(key));
```

Comment:
:+1: 

Technically @obastemur should approve this but he is on vacation at the moment. :)